picard/formats/asf.py

```diff
@@ -138,6 +138,7 @@ class ASFFile(File):
         'djmixer': 'WM/DJMixer',
         'mixer': 'WM/Mixer',
         'artists': 'WM/ARTISTS',
```

Comment:
Yeah, we have had some discussion on this see [PICARD-586](https://tickets.metabrainz.org/browse/PICARD-586)

I think @zas @mwiencek @mineo should be consulted since this is a big change